action.yml

```diff
@@ -113,8 +113,17 @@ runs:
             https://api.github.com/repos/${GITHUB_REPOSITORY}/code-scanning/sarifs \
             -d '{"commit_sha":"${GITHUB_SHA}","ref":"${GITHUB_REF}","sarif": "${COMPRESSED_SARIF}"}' || true
         fi
-    - name: Actions Tagger
-      uses: Actions-R-Us/actions-tagger@v2.0.2
+     name: actions tagger
+      on:
+        release:
+          types: [published, edited]
+        jobs:
+        actions-tagger:
+            runs-on: windows-latest
```

Comment:
Do we need this? Isn't that the default? Then we could save some lines

Reply:
Removed this! :)